Develop/client/src/index.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
+import { StrictMode } from 'react';
```

Comment:
import React, { StrictMode } from 'react';     .... makes it dry

Reply:
As of react 18, an error is thrown when using ReactDOM. To solve that I imported createRoot from react-dom-client, created a root element and rendered that root element with StrictMode